<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sqlite</class>
 <widget class="QWidget" name="Sqlite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="datetimeedit_1">
       <property name="date">
        <date>
         <year>2025</year>
         <month>7</month>
         <day>17</day>
        </date>
       </property>
       <property name="time">
        <time>
         <hour>9</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="datetimeedit_2">
       <property name="date">
        <date>
         <year>2025</year>
         <month>12</month>
         <day>25</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PbSearchDB">
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PbDeleteDB">
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5">
     <item>
      <widget class="QTableWidget" name="PbClear">
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>날짜</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>조도</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>온도</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>습도</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="9">
       <item>
        <layout class="QHBoxLayout" name="ChartViewLayout"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
